packages/frontend/app/components/LiquidityDialog.tsx

```diff
@@ -45,7 +55,12 @@ export const LiquidityDialog = ({
                     type='number'
                     name='amount'
                     label='Amount'
+                    onChange={e => setAmount(Number(e.target.value))}
                   />
+                  <div className='mt-2'>
+                    <p>Based on the asset:</p>
+                    <p>Amount {amount} = {amount / Math.pow(10, asset.scale)} {asset.code} </p>
```

Comment:
I think this could be simplified a bit. I don't think we need to restate the amount since its in the input nor that it's translating to the current asset scale because of the context.

I think this would be fine:

```suggestion
                    <p>
                      {amount / Math.pow(10, asset.scale)} {asset.code}{' '}
                    </p>
```
![image](https://github.com/user-attachments/assets/53b19a46-4989-463e-a0b3-14eaade4eed9)

But perhaps a dummy input field for the asset scale really drives home the point (user can see 0 and 100 => 100, 2 and 100 => 1, etc.):

```tsx
                  <Input
                    required
                    disabled
                    type='number'
                    label='Asset Scale'
                    value={asset.scale}
                  />
```
![image](https://github.com/user-attachments/assets/5f09ebf3-894f-410f-afdb-60c0c58c1104)